### PR TITLE
This is script for creating new collection for the yelp and hygiene j…

### PR DESCRIPTION
**Pull request by psk285**

### CODE REVIEW

File: utils/collections/yelp_join_hygiene.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from pymongo import MongoClient
```
Comment (rj1061) [inline utils/collections/yelp_join_hygiene.py]: You don't need this if you are importing * later

File: utils/collections/yelp_join_hygiene.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+from pymongo import *
```
Comment (rj1061) [inline utils/collections/yelp_join_hygiene.py]: It is not a good practice to import \* in python. Takes longer to interpret.

File: utils/collections/yelp_join_hygiene.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+client = MongoClient('localhost', 27017)
+db = client.weisheng
+test = {"test":"test"}
+collection = db.yelp_join_hygiene.insert(test)
+collection = db.yelp_join_hygiene.remove({})
+
```
Comment (rj1061) [inline utils/collections/yelp_join_hygiene.py]: Why the extra lines?

**Comment by rj1061** [comment]
LGTM :+1:

File: utils/collections/yelp_join_hygiene.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+from pymongo import MongoClient
+client = MongoClient('localhost', 27017)
+db = client.weisheng
+test = {"test":"test"}
+collection = db.yelp_join_hygiene.insert(test)
```
Comment (maqzi) [inline utils/collections/yelp_join_hygiene.py]: no need to assign it to anything

**Comment by muraleo** [comment]
Looks good~

**Pull request merged**